SensorNode/node.py

```diff
@@ -79,10 +79,14 @@ def __init__(self, config_path: str):
                 self.data_endpoint = socket.gethostbyname(endpoint)
             except:
                 time.sleep(1)
+
+        
         uuid_str = self._config_tree['data_server_uuid']
         self.data_server_uuid = uuid.UUID(uuid_str)
         self.port_number = int(self._config_tree['port'])
-        self._log.info(f"Resolving endpoint as {self.data_endpoint}, UUID: {self.data_server_uuid}")
+        self._log.info(f"Resolving endpoint as {self.data_endpoint}, UUID: {self.data_server_uuid}, port: {self.port_number}")
```

Comment:
Use lazy % formatting in logging functions pylint(logging-fstring-interpolation)

Reply:
Line too long (126/100) pylint(line-too-long)